sim/mage/arcane_missiles.go

```diff
@@ -35,7 +35,7 @@ func (mage *Mage) registerArcaneMissilesSpell() {
 
 		Cast: core.CastConfig{
 			DefaultCast: core.Cast{
-				Cost: baseCost,
+				Cost: (1 - .01*float64(mage.Talents.ArcaneFocus)),
```

Comment:
This looks very wrong  :)

Reply:
oh lol without the base. Is there a cleaner way to do reduced cost on all arcane spells also btw? I mean I guess this works fine if I actually have the base value.

Reply:
One neat way of doing it which I've thought about but never tried: use the unit.Spellbook array to loop over all the registered spells, check their school, and then apply it. You might need to be careful about initialization order of things though.